redhat/squeezeboxserver.init

```diff
@@ -164,6 +164,13 @@ else
 	exit 1
 fi
 
+# Do not change this, it will most likely break the squeezeboxserver
+# installation. The directory /usr/lib/perl5/vendor_perl is where the RPM
+# package drops the Slim perl modules. Setting PERL5LIB to this location will
+# make sure that the squeezebox server executable looks here before it scans the
+# directories in the perl @INC array.
+export PERL5LIB="/usr/lib/perl5/vendor_perl"
```

Comment:
Shouldn't you add this path to `PERL5LIB`, rather than replace it completely? The user could potentially have it set for good reasons.